packages/suite-build/configs/landing.webpack.config.ts

```diff
@@ -0,0 +1,53 @@
+import path from 'path';
+import webpack from 'webpack';
+import CopyPlugin from 'copy-webpack-plugin';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import ReactRefreshWebpackPlugin from '@pmmmwh/react-refresh-webpack-plugin';
+
+import routes from '../../suite/src/config/suite/routes';
```

Comment:
those are suite routes not landing page routes (landing page has only two routes so far `/` and `/start`)

Reply:
14a42c2